### ISSUE TITLE
cleos system newaccount ERROR: ExtrasError: [EOS5xxxxx]

### ISSUE THREAD

**Issue by unqjohnny** [EOSIO/eos#4565]
I want to create new account
'''
cleos system newaccount  --stake-net '0.0001 EOS' --stake-cpu '0.001 EOS' —buy-ram-kbytes 3 g4ytgnbtgage  johnaccount EOS5xxxxx
'''
but return:
'''
ERROR: ExtrasError: [EOS5xxxxx]
Create an account, buy ram, stake for bandwidth for the account
Usage: /opt/eosio/bin/cleos system newaccount [OPTIONS] creator name OwnerKey [ActiveKey]
'''
How can i solve this?

**Comment by dhaneshvb** [comment]
Is the owner key valid?
Just FYI : the usage is as follows:

sage: cleos system newaccount [OPTIONS] creator name OwnerKey [ActiveKey]

Positionals:
  creator TEXT                The name of the account creating the new account (required)
  name TEXT                   The name of the new account (required)
  OwnerKey TEXT               The owner public key for the new account (required)
  ActiveKey TEXT              The active public key for the new account

Options:
  -h,--help                   Print this help message and exit
  --stake-net TEXT            The amount of EOS delegated for net bandwidth (required)
  --stake-cpu TEXT            The amount of EOS delegated for CPU bandwidth (required)
  --buy-ram-kbytes UINT       The amount of RAM bytes to purchase for the new account in kibibytes (KiB), default is 8 KiB
  --buy-ram TEXT              The amount of RAM bytes to purchase for the new account in EOS
  --transfer                  Transfer voting power and right to unstake EOS to receiver
  -x,--expiration             set the time in seconds before a transaction expires, defaults to 30s
  -f,--force-unique           force the transaction to be unique. this will consume extra bandwidth and remove any protections against accidently issuing the same transaction multiple times
  -s,--skip-sign              Specify if unlocked wallet keys should be used to sign transaction
  -j,--json                   print result as json
  -d,--dont-broadcast         don't broadcast transaction to the network (just print to stdout)
  -r,--ref-block TEXT         set the reference block num or block id used for TAPOS (Transaction as Proof-of-Stake)
  -p,--permission TEXT ...    An account and permission level to authorize, as in 'account@permission'
  --max-cpu-usage-ms UINT     set an upper limit on the milliseconds of cpu usage budget, for the execution of the transaction (defaults to 0 which means no limit)
  --max-net-usage UINT        set an upper limit on the net usage budget, in bytes, for the transaction (defaults to 0 which means no limit)

**Comment by taokayan** [comment]
Your command has some issues:
- All EOS amount should have exactly 4 decimals: ```0.001 EOS``` should be ```0.0010 EOS```
- I doubt that 3kb is not enough to create one account. 4kb is more acceptable.

**Comment by unqjohnny** [comment]
@dhaneshvb  @taokayan , thans for your replay.
'''
1、cleos wallet keys,return:
[
  "EOS53cJrAucxiz4d9hUXNn9Ss4CowoFFmtzxDutyC4SfVwNbsFiT6",
  "EOS7iFEsqjwRjm9jkzDqYSZrd3rf5a7supZ8wUzyNZtwPFk8DhUMF"
]

2、cleos get accounts EOS7iFEsqjwRjm9jkzDqYSZrd3rf5a7supZ8wUzyNZtwPFk8DhUMF， return:
{
  "account_names": [
    "g4ytgnbtgage"
  ]
}

3、
And the key:EOS53cJrAucxiz4d9hUXNn9Ss4CowoFFmtzxDutyC4SfVwNbsFiT6  is created by cleos create key. 
then use command:
cleos system newaccount  --stake-net '0.0001 EOS' --stake-cpu '0.0010 EOS' —buy-ram-kbytes 4  g4ytgnbtgage  tjohnaccount EOS53cJrAucxiz4d9hUXNn9Ss4CowoFFmtzxDutyC4SfVwNbsFiT6

return:
ERROR: ExtrasError: [EOS53cJrAucxiz4d9hUXNn9Ss4CowoFFmtzxDutyC4SfVwNbsFiT6]
Create an account, buy ram, stake for bandwidth for the account
Usage: /opt/eosio/bin/cleos system newaccount [OPTIONS] creator name OwnerKey [ActiveKey]
'''

**Comment by taokayan** [comment]
looks like you missed a ```-``` in ```—buy-ram-kbytes```

**Comment by taokayan** [comment]
is your cleos connecting to the slow server node? normally it should be expired. try to connect to some block producer node, like ```-u https://eu1.eosdac.io:443```

**Comment by unqjohnny** [comment]
I connect to https://eu1.eosdac.io:443 now. but now new error:
```
1656380ms thread-0   main.cpp:429                  create_action        ] result: {"binargs":"xxx"} arg: {"code":"eosio","action":"buyrambytes","args":{"payer":"g4ytgnbtgage","receiver":"tjohnaccoun1","bytes":3072}} 
1658677ms thread-0   main.cpp:429                  create_action        ] result: {"binargs":"xxx"} arg: {"code":"eosio","action":"delegatebw","args":{"from":"g4ytgnbtgage","receiver":"tjohnaccoun1","stake_net_quantity":"0.0001 EOS","stake_cpu_quantity":"0.0010 EOS","transfer":false}} 
Error 3050003: eosio_assert_message assertion failure
```

and this is my account info below:
```
cleos2 get account  g4ytgnbtgage: 
permissions: 
     owner     1:    1 EOS7iFEsqjwRjm9jkzDqYSZrd3rf5a7supZ8wUzyNZtwPFk8DhUMF
        active     1:    1 EOS7iFEsqjwRjm9jkzDqYSZrd3rf5a7supZ8wUzyNZtwPFk8DhUMF
memory: 
     quota:     7.955 KiB    used:     3.365 KiB  

net bandwidth: 
     staked:          0.9800 EOS           (total stake delegated from account to self)
     delegated:       0.0000 EOS           (total staked delegated to account from others)
     used:                 0 bytes
     available:          520 KiB  
     limit:              520 KiB  

cpu bandwidth:
     staked:          0.9800 EOS           (total stake delegated from account to self)
     delegated:       0.0000 EOS           (total staked delegated to account from others)
     used:                 0 us   
     available:        101.4 ms   
     limit:            101.4 ms   

producers:     <not voted>
```
Is there any problem with my account?? @taokayan

**Comment by taokayan** [comment]
most likely the creator account doesn't have enough liquid balance to buy 3kB RAM.

```
./cleos -u https://eu1.eosdac.io:443 get account g4ytgnbtgage
permissions: 
     owner     1:    1 EOS7iFEsqjwRjm9jkzDqYSZrd3rf5a7supZ8wUzyNZtwPFk8DhUMF
        active     1:    1 EOS7iFEsqjwRjm9jkzDqYSZrd3rf5a7supZ8wUzyNZtwPFk8DhUMF
memory: 
     quota:     7.955 KiB    used:     3.365 KiB  

net bandwidth: 
     staked:          0.9800 EOS           (total stake delegated from account to self)
     delegated:       0.0000 EOS           (total staked delegated to account from others)
     used:                 0 bytes
     available:          520 KiB  
     limit:              520 KiB  

cpu bandwidth:
     staked:          0.9800 EOS           (total stake delegated from account to self)
     delegated:       0.0000 EOS           (total staked delegated to account from others)
     used:                 0 us   
     available:        101.4 ms   
     limit:            101.4 ms   

EOS balances: 
     liquid:            0.1000 EOS
     staked:            1.9600 EOS
     unstaking:         0.0000 EOS
     total:             2.0600 EOS
```
